Source/WebKit/UIProcess/ProcessThrottler.cpp

```diff
@@ -52,16 +52,28 @@ ProcessThrottler::~ProcessThrottler()
     invalidateAllActivities();
 }
 
-void ProcessThrottler::addActivity(ForegroundActivity& activity)
+bool ProcessThrottler::addActivity(ForegroundActivity& activity)
 {
+    if (!m_allowActivity) {
+        PROCESSTHROTTLER_RELEASE_LOG("addActivity: not allowed to add foreground activity %s", activity.name().characters());
```

Comment:
I recommend we make Activity::isQuietActivity() public and only do this logging if the activity is not quiet, as mentioned in my earlier review. Otherwise, the logging is going to be very noisy due to IPC and would provide a useful activity name anyway.

Reply:
oh I thought "when activity is quiet" means "when activity is not allowed"; turns out there's isQuietActivity. Will change.